specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1984,6 +2272,88 @@
         }
       }
     },
+    "SqlServerInstanceProperties": {
+      "description": "Properties of SqlServerInstance.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "null"
+        },
+        "edition": {
+          "type": "string",
+          "description": "null"
+        },
+        "containerResourceId": {
+          "type": "string",
+          "description": "null"
+        },
+        "createTime": {
+          "type": "string",
+          "description": "null"
+        },
+        "updateTime": {
+          "type": "string",
+          "description": "null"
+        },
```

Comment:
these should be part of the top-level systemData contract for new resources.  Details here:  [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources)

Reply:
This new resource type shared the common code with existing resource types of this resource provider. systemData contract, as a 2020 new requirement, is not added into the common infra of this resource provider yet. Consider this new resource type is for private preview purpose, is it ok to be an exception? Otherwise, all published existing resource types of this resource provider share the same issue too.

Reply:
Its not a mandate yet so yes, its technically OK.  You should aim to have it implemented before you leave private preview